pom.xml

```diff
@@ -18,7 +18,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <guava.version>11.0.2</guava.version>
+        <guava.version>19.0</guava.version>
```

Comment:
We did get compatibility problems, yes. @danielnorberg you picked this version, IIRC, any opinions? @lburgazzoli, what are the Guava versions that are OSGI ready?